modules/local/nf-core-modified/medaka/main.nf

```diff
@@ -0,0 +1,42 @@
+process MEDAKA {
```

Comment:
What are the modications made, so I can review accordingly.

Reply:
apologies forgot to put the comment of the modification at the top - I had to add a line that unzips the assembly file because this won't take in `.gz` files for that